src/components/revo-grid/revo-grid.tsx

```diff
@@ -59,6 +59,8 @@ export class RevoGridComponent {
   @Prop() resize: boolean = false;
   /** When true cell focus appear. */
   @Prop() canFocus: boolean = true;
+  /** When true enable clipboard. */
+  @Prop() useClipboard: boolean = false;
```

Comment:
We need to make it true by default, 'coz other users who used to have it enabled will suffer from it.